nist/int.go

```diff
@@ -346,6 +341,21 @@ func (i *Int) BigEndian(min, max int) []byte {
 	return buf
 }
 
+// Set value to a number represented by a byte string.
+// Endianness depends on the endianess set in i.
+func (i *Int) SetBytes(a []byte) *Int {
+	i.V.SetBytes(a).Mod(&i.V, i.M)
+	return i
+}
+
+// Set value to a number represented in a little-endian byte string.
+func (i *Int) SetLittleEndian(a []byte) *Int {
```

Comment:
I probably meant that I want to keep the *ability* to get little-endian behavior out of nist.Int - but that doesn't mean we need to keep the SetLittleEndian *method*. :)